package.json

```diff
@@ -117,7 +124,7 @@
     "lerna": "^3.16.4",
     "link-parent-bin": "^1.0.0",
     "loader-runner": "^4.1.0",
-    "postcss": "^8.2.10",
+    "postcss": "8.3.11",
```

Comment:
Css linter is failing due to latest postcss release @8.4.1 so hardcoding this to 8.3.11 for the time being.